sdk/purview/Azure.Analytics.Purview.Workflows/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "CancellationToken can be passed through RequestOptions")]
```

Comment:
Why we need this suppression?

Reply:
It is generated automatically

Reply:
It is not generated. All the generated stuff will be under Gnerated folder.

Reply:
Fixed, thanks